project1/css/index.css

```diff
@@ -28,6 +28,93 @@ article, aside, details, figcaption, figure,
 footer, header, hgroup, menu, nav, section {
 	display: block;
 }
+
+.container header{
+  display: flex;
+  flex-direction: row;  
+  justify-content: space-between;
+  width: 1150px;
+}
+.container header ul{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 800px;
+  }
+.container a{
+  margin-left: 30px;
+  text-decoration: none;
+  color: gray;
+}
+
+.cta{
+  display: flex;
+  justify-content: space-around;
+  margin-top:40px;
+  width:1250px;
+  margin-bottom: 70px; 
+}
+.cta-text{
+  font-size: 80px;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-flow: column;
+  height: 300px;
+  margin-top: 30px;
+  align-items: center;
+  }
+.cta-text button{
+  background: white;
+  width: 180px;
+  height:40px;
+  font-size: 19px;
+  border: 1.5px solid black;
+  font-family: Titillium Web;
+ }
+
+.main-content{
+ 
+  width:1100px;
+  margin: 50px;
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;
+  line-height: 18 px;
+}
+.main-content .top-content{
+  display: flex;
+  justify-content: space-around;
+  margin-top: 50px;
+}
+.middle-img{
+  margin-top:30px;
+  margin-bottom:30px;
+  width: 100%;  
+}
+.bottom-content{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
+}
+
```

Comment:
You should either have line breaks between every code block, or no line breaks at all. Gotta stay consistent!